dsgrid/cli/registry.py

```diff
@@ -1500,13 +1534,15 @@ def update_dataset(
     dataset_config_file: Path,
     dataset_id: str,
     log_message: str,
-    dataset_path: Path | None,
     update_type: VersionUpdateType,
     version: str,
 ):
     """Update an existing dataset in the registry. The contents of the JSON/JSON5 file
     must match the data model defined by this documentation:
     https://dsgrid.github.io/dsgrid/reference/data_models/dataset.html#dsgrid.config.dataset_config.DatasetConfigModel
+
+    If the config file includes a UserDatasetSchema with file paths, the data will be
+    re-read from those paths. Otherwise, the existing data in the registry is used.
```

Comment:
Is this still accurate?

Reply:
yes. But I'm not sure how well the update process really works. We don't really use it for datasets. We typically delete and re-register.